azure-pipelines.yml

```diff
@@ -387,28 +387,28 @@ stages:
       displayName: 'Run ITs'
       strategy:
         matrix:
-          vs2017_dev:
-            SQ_VERSION: "LATEST_RELEASE[8.0]"
+          vs2017_latest79:
```

Comment:
I'm not sure if we need a full test matrix w/ all SQ flavors for all VS flavors.

I would keep a "sparse" matrix which would be good enough to catch errors, something like

- vs2017_latest89
- vs2019_latest79
- vs2022_latest89
- vs2022_dev

I am happy to be contradicted, however I'm not sure that the gain of running one-to-one tests is worth the resources.

Because the integration with MSBuild versions should not be coupled at all with SQ versions, AFAIK.

@duncanp-sonar WDYT?

Reply:
@andrei-epure-sonarsource yes, the MSBuild version and SQ version should be independent, so a sparse matrix would do.

Reply:
> 
vs2017_latest89
vs2019_latest79
or
vs2017_latest79
vs2019_latest89?

Reply:
IMO 
vs2017_latest89
vs2019_latest79

to avoid suggesting from the matrix that there would be a connection beetween old MSB and old SQ

Reply:
>If I understood it correctly during the scanner monthly meeting we decided to have this full test matrix until @tom-howlett-sonarsource comes back with the clarified support policy. 

The decision was to test old SQ versions as well, but AFAIK we didn't talk about testing each SQ with each MSB versions